doc/release/1.17.0-notes.rst

```diff
@@ -325,41 +359,39 @@ Error messages from array comparison tests such as
 This information makes it easier to update absolute and relative error
 tolerances.
 
-Replacement of the `fftpack`-based FFT module by the `pocketfft` library
-------------------------------------------------------------------------
-Both implementations have the same ancestor (Fortran77 `FFTPACK` by Paul N.
-Swarztrauber), but `pocketfft` contains additional modifications which
-improve both accuracy and performance in some circumstances. For FFT lengths
-containing large prime factors, `pocketfft` uses Bluestein's algorithm, which
-maintains `O(N log N)` run time complexity instead of deteriorating towards
-`O(N*N)` for prime lengths. Also, accuracy for real-valued FFTs with near-prime
-lengths has improved and is on par with complex-valued FFTs.
+Replacement of the fftpack based FFT module by the pocketfft library
+--------------------------------------------------------------------
+Both implementations have the same ancestor (Fortran77 FFTPACK by Paul N.
+Swarztrauber), but pocketfft contains additional modifications which improve
+both accuracy and performance in some circumstances. For FFT lengths containing
+large prime factors, pocketfft uses Bluestein's algorithm, which maintains
+``O(N log N)`` run time complexity instead of deteriorating towards ``O(N*N)``
+for prime lengths. Also, accuracy for real valued FFTs with near prime lengths
+has improved and is on par with complex valued FFTs.
 
 Performance improvements for integer sorts
 ------------------------------------------
-
 ``sort``, ``argsort``, ``ndarray.sort`` and ``ndarray.argsort`` now use radix
 sort as the default stable sort for integers and booleans. This is faster than
```

Comment:
I thought this was only true for small integer/boolean dtypes? More generally, I  think the notes on timsort/radix sort should  probably be merged.